### PR TITLE
add new starter as dependabot reviewer

### PR DESCRIPTION
**Pull request by jsugarman**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -38,6 +38,7 @@ updates:
   - "jsugarman"
   - "kmahern"
   - "jrmhaig"
+  - "mabley"
```
Comment (mabely) [inline .github/dependabot.yml]: There's just a small typo in the username :) - "mabely"

Comment (jsugarman) [reply .github/dependabot.yml]: oops, sorry

Comment (mabely) [reply .github/dependabot.yml]: No worries

Comment (jsugarman) [reply .github/dependabot.yml]: fixed now

**Review by mabely — APPROVED** [review]
Looks good, and thanks for this

**Pull request merged**